### DISCUSSION TITLE
`GetPackageOriginalFile` optional destination path

### DISCUSSION THREAD

**Discussion by Mysak0CZ** [CyberShadow/aconfmgr#157] [Ideas]
It would be useful for `GetPackageOriginalFile` to have optional third argument - destination path - used instead of the original path from the package.

Why would this be useful?
Fair few packages (the example that prompted me to open this suggestion is pipewire) have default configuration files in `usr` folder which you can modify by copying it to the `etc` folder and edit it there. For making small, inlined change to the original file, it would be useful to be able to put it into new location directly, instead of extracting it temporarily and then using `CreateFile`.

**Comment by CyberShadow** [comment]
So I understand that the goal is to be able to write
```shell
f=$(GetPackageOriginalFile some-package /var/lib/foo.conf /etc/foo.conf)
```
instead of
```shell
f=$(CreateFile /etc/foo.conf)
mv "$(GetPackageOriginalFile some-package /var/lib/foo.conf)" "$f"
```

Seems fine to me, feel free to submit a PR to add this.
